openapi/components/schemas/RiskMetadata.yaml

```
@@ -234,6 +234,10 @@ properties:
     description: Number of transactions for this billing address in the last 24 hours.
     type: integer
     readOnly: true
+  paymentInstrumentApprovedTransactionCount:
+    description: Number of approved transactions for this payment instrument.
```

Minor note: perhaps `the` instead of `this` @noon-dawg

```suggestion
    description: Number of approved transactions for the payment instrument.
```

---

Well spotted @nanobash!
Probably best to use `this` here so that it is consistent with the fields above it